Description: Remove obsolete systemd targets
 syslog.target is now socket-activated and hence obsolete
 Patch was submitted upstream but rejected. Software runs
 primarily on embedded systems with older software. Newer
 systems do not require the patch.
Author: Felix Lechner <felix.lechner@lease-up.com>
Forwarded: yes, but rejected
Last-Update: 2017-12-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/svxlink/systemd/remotetrx.service.in
+++ b/src/svxlink/systemd/remotetrx.service.in
@@ -20,7 +20,7 @@
 Description=SvxLink remote transceiver repeater control software
 Documentation=man:remotetrx(1)
 Requires=svxlink_gpio_setup.service
-After=network.target remote-fs.target syslog.target time.target
+After=network.target remote-fs.target time.target
 After=svxlink_gpio_setup.service
 
 [Service]
--- a/src/svxlink/systemd/svxlink.service.in
+++ b/src/svxlink/systemd/svxlink.service.in
@@ -20,7 +20,7 @@
 Description=SvxLink repeater control software
 Documentation=man:svxlink(1)
 Requires=svxlink_gpio_setup.service
-After=network.target remote-fs.target syslog.target time.target
+After=network.target remote-fs.target time.target
 After=svxlink_gpio_setup.service
 
 [Service]
--- a/src/svxlink/systemd/svxreflector.service.in
+++ b/src/svxlink/systemd/svxreflector.service.in
@@ -19,7 +19,7 @@
 [Unit]
 Description=SvxReflector SvxLink node linking software
 Documentation=man:svxreflector(1)
-After=network.target remote-fs.target syslog.target time.target
+After=network.target remote-fs.target time.target
 
 [Service]
 EnvironmentFile=@SYSTEMD_DEFAULTS_FILES_DIR@/svxreflector
